parrot/src/pfs_service_local.cc

```diff
@@ -153,25 +149,6 @@ process to sleep and wait for actual input to become ready.
 		result = ::read(fd,data,length);
 		if(result>0) last_offset = offset+result;
 
-		if(result==0 && offset==0) {
```

Comment:
The reason for this roundabout code is explained by the comment above.  Does that reason no longer apply?